openlibrary/plugins/importapi/tests/test_import_validator.py

```diff
@@ -18,7 +25,27 @@
 
 def test_validate():
     assert validator.validate(valid_values) is True
+
```

Comment:
```suggestion


@pytest.mark.parametrize('field', ["title", "source_records", "authors", "publishers", "publish_date"])
def test_validate_record_with_missing_required_fields(field):
    invalid_values = valid_values.copy()
    del invalid_values[field]
    with pytest.raises(ValidationError):
        validator.validate(invalid_values)

```
I think that we should also test for the cases where a required field is not in the import record.  This test should do that.

I think that this was what I was trying to achieve with the `del invalid_values['authors']` line in the old test case.